packages/greenbox-web/README.md

```diff
@@ -0,0 +1,32 @@
+
+# greenbox-web
+
+```
+packages
+...
+└── greenbox-web
+    ├── src
+        ├── components
+        |   └── [various tsx helpers]
+        └── store
+```
+
+This is by far the most out-of-the-ordinary script for most KOL scripters, as it is built in React and is staged via a Github-Pages deployment. It is not immediately straightforward what things are doing if you do not know React. This is the script that builds the website visible on [greenbox.loathers.net](https://greenbox.loathers.net). There are two primary folders you'll need to concern yourself with if you intend to make a PR for greenbox that touches **greenbox-web**.
+
+Within `greenbox-web/src/components`, you will find a whole host of grids, functions, and builders that are used to help build the individual components of the website. Every file in `components` is a TSX file, which is a TypeScript file written in JSX, an embeddable XML-like syntax used commonly in React applications. For example, at writing, the `Familiars.tsx` file contains (effectively) the entirety of the Familiars dropdown that exists on the web deployment. It imports a bunch of data from **greenbox-data**, stages the data according to the overarching style created in `greenbox-web/src/store`, and creates the general formatting for the Familiars box. 
+
+The same is true of many of the items within components, but not all. For instance, `AlphaImage.tsx` is a helper function built by Gausie that essentially builds transparency where none exists for the black-and-white images KOL is known for, allowing for polyfill of the whitespace when you are filling a box green for the web interface. `Image.tsx` simply redirects images to KOL's image server, allowing us to glide on the coattails of TPTB's web hosting for the six people in the known universe who use our script. The components have a lot of different elemnts that are bespoke and relevant only to that specific component. Please dig through these and attempt to understand their working patterns before modifying them.
+
+Within `greenbox-web/src/store`, you will find the .ts files that are used to combine all the myriad components into a final website. The primary file here is (fittingly) `index.ts`, which imports essentially everything from `components` and plops them all down into a file that users can view. 
```

Comment:
This isn't quite right - the store folder manages the sort of shared brain of all of the components - our application state

Reply:
The enploppening happens in main.tsx